doc/ch-nudge_desc.rst

```diff
@@ -0,0 +1,103 @@
+Synopsis
+========
+
+::
+
+   $ ch-nudge [OPTIONS] ACTION IMAGE_REF
+
+Description
+===========
+
+.. warning::
+
+   This script is experimental. Please report the bugs you find so we can fix
+   them!
+
+.. note::
+   This script only works with images built with ch-grow stored in
+   CH_GROW_STORAGE.
+
+Delete, list, or push image data by HTTPS to repository via :code:`IMAGE_REF`.
+Where the image reference :code:`IMAGE_REF` is loosely defind as a string
+consisting of a hostname, path, and image. For example,
+:code:`HOSTNAME[:PORT]/PATH/[TO/,]/IMAGE[:TAG])`. See examples below and the FAQ
+section for more details.
+
+.. note::
+   See examples below and FAQ for more details.
+
+Note this script is typically not used directly. Most tasks instead use the
+underlying code with :code:`ch-grow(1)`.
+
+:code:`ACTION` must be one of the following:
+
+  :code:`delete IMAGE_REF`
+    TODO
+  :code:`list IMAGE_REF`
+    TODO
+  :code:`push IMAGE_REF`
+    TODO
```

Comment:
I wonder if we should also consider this interface:

```
$ ch-nudge --delete IMAGE_REF
$ ch-nudge --list IMAGE_REF
$ ch-nudge IMAGE_REF [DEST_IMAGE_REF]
```

i.e.:

1. default is push
2. would need to document that list should not have a tag or hash